src/averageAges.js

```diff
@@ -20,6 +20,21 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const addAge = people.map(item =>
+    Object.assign(item, { age: (item.died - item.born) }, {
+      centuries:
+        Math.ceil(item.died / 100),
+    }));
+  let filSexDied = [];
+
+  century
+    ? filSexDied = addAge.filter(item => item.sex === 'm'
+    && item.centuries === century)
+    : filSexDied = addAge.filter(item => item.sex === 'm');
```

Comment:
```suggestion
  let filSexDied = addAge.filter(item =>
    century
      ? item.sex === 'm' && item.centuries === century
      : item.sex === 'm'
    );
```
Це можна було б записати простіше.
Або ще простіше (без змінної addAge):
```
  const men = people.filter(person =>
    century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : (person.sex === 'm')
  );
```